src/components/sidebar/index.tsx

```diff
@@ -131,6 +136,27 @@ const Sidebar = ({
 				showFilterInput={false}
 			/>
 		)
+	} else if (isProductPanel) {
+		sidebarContent = (
+			<>
+				<NavigationMenu.Root>
+					<ProductPanel
+						productCategories={mobileNavigationData}
+						promo={navPromo}
+						sidePanel={sidePanelContent}
+					/>
+				</NavigationMenu.Root>
+				{showResourcesList && (
+					<SidebarNavList className={s.mainMenuResources}>
```

Comment:
Uses just the `<SidebarNavList />` markup due to the following needs:
- Need to adjust padding via `className`
- Need to go without a horizontal rule above the list

Reply:
Can you add more padding to the bottom of the resources section? Currently, on mobile the Terraform Registry link is barely clickable because of the searchbar and phone controls cover most of it (at least for me on a samsung device) .

Reply:
Ah, good point! 

I noticed this was an issue on all mobile menus, not just those with resource sections 🫠 

To solve, I changed the `100vh` declaration in the mobile menu container's height to `100%` here:
0da6be4

However, despite in-code comments to the contrary, that change affected the desktop sidebar as well 🤔 So I made sure it only affected mobile via this commit:
c9a4ae2